File: Chapter 04/JS/section_4_4_anti_patterns/after_rich_domain_thin_controller/application/orderResponse.js
class OrderResponse {
/**
 * THE RESPONSE DTO
 * ARCHITECTURE NOTE: Returns the atomic truth calculated by the domain model.
 */
    constructor(orderId = 0, totalPrice = 0.0, customerEmail = "") {
        this.orderId = orderId;
        this.totalPrice = totalPrice;
        this.customerEmail = customerEmail;
    }
}
    module.exports = OrderResponse;

File: Chapter 04/Python/section_4_4_anti_patterns/after_rich_domain_thin_controller/application/order_response.py

class OrderResponse:
    """
    DTO for the order creation response.
    """
    def __init__(self, order_id: int = 0, total_price: float = 0.0, customer_email: str = ""):
        self.order_id = order_id
        self.total_price = total_price
        self.customer_email = customer_email